docs/ensembl-help/using-ensembl/ensembl-apps/species-selector/species-selector-intro.md

```diff
@@ -11,20 +11,45 @@ status: draft
 
 # Species selector
 
-The Species selector allows you to view and select the species and genome assemblies that you wish to view in the Genome browser and Entity viewer.
+The Species selector allows you to view and select the species genomes and assemblies that you wish to view in the Genome browser and Entity viewer.
 
 ## Selecting a species
 
-In order to start using the Genome browser and Entity viewer, you first need to select a species and genome assembly. There are two main ways to select a species:
+In order to start using the Genome browser and Entity viewer, you need to select a species genome and assembly. 
 
-- Click on the icons representing our 42 most popular species and click the green 'Add' button.
-- Type the name of a species or genome assembly into the search box and select from the auto-complete options in the drop-down menu. If there is more than one genome assembly available for your species of interest, a drop-down menu will appear allowing you to select your genome assembly of interest. Click the green 'Add' button to add the selected species and genome assembly.
+There are two ways to select a species genome:
+
+1.  Select from the 42 most popular species genomes
+
+> Select the icon representing your species genome of interest from the 42 most popular species genomes and select the green 'Add' button.
```

Comment:
Expectation:
"from an initial selection of 7 selected popular species genomes... In the future this will be extended to include the 42 most popular species genomes'

Reply:
> from an initial selection of 7 selected popular species genomes...

This would mean that we'll have to remember to update this text when we more species. If instead of specifying the number 7, you write something like "icons of species currently available on the site are marked in blue", this will remain true regardless of how many popular species we have on the site at any given moment.

Reply:
The help will have to be continually updated, we need to be as informative as possible for users.

Reply:
done